contracts/prize-strategy/multiple-winners/MultipleWinners.sol

```diff
@@ -1,18 +1,29 @@
 // SPDX-License-Identifier: MIT
-
 pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
 
 import "../PeriodicPrizeStrategy.sol";
 
 contract MultipleWinners is PeriodicPrizeStrategy {
 
+  struct MultipleWinnersPrizeSplit {
+      address target;
+      uint16 percentage;
+  }
+
   uint256 internal __numberOfWinners;
+  
+  MultipleWinnersPrizeSplit[2] internal _prizeSplits;
```

Comment:
Generally I've only seen be a single underscore to represent an internal variable. But to match __numberOfWinners it might be worthwhile to add the additional underscore.

No. When setting the length it doesn't start from the 0 index. So it's only a length of 2.

After speaking with Aodghan last week we decided to remove the fixed length array all together, since it only added unnecessary restrictions with the current approach.